### PR TITLE
Fix BE scraper and improve matching

### LINKED ISSUE #324: BE scraper fails
[BE scraper fails](https://github.com/openZH/covid_19/actions/runs/66489667):

```
Run the scraper...
Error: Not matched input line: BE -                      -     826 FAILED 2020-03-30T12:16:55+00:00
# URLs: https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html
# Errors: Missing date. Missing cases. Date and time is None. Confirmed cases is None
```

### PR DESCRIPTION
**Pull request by calmyournerves**
Fixes #324 

Fixes the BE scraper. Also updated it to check the data structure using regular expressions.

```
./scrape_be.sh                                                                                                                                                                                                                    
BE
Downloading: https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html
Scraped at: 2020-03-30T15:54:39+02:00
Date and time: 30.03.2020
Confirmed cases: 826
Deaths: 13
Hospitalized: 112
ICU: 21
Vent: 17
```

### CODE REVIEW

File: scrapers/scrape_be.sh [hunk 1]
```diff
@@ -1,39 +1,91 @@
...
-print('Date and time:', sc.find(r'Stand: (.+)\)', d))
-# This isn't really the best method, but it works for now.
-print('Confirmed cases:', sc.find(r'<td .*<strong>([0-9]+)<', d))
-print('Deaths:', sc.find(r'<td[^<>]*>([0-9]+)</td>', d))
+m = re.search(r'<table .*? summary="Laufend aktualisierte Zahlen zu den Corona-Erkrankungen im Kanton Bern">(.*?)<\/table>', d, flags=re.MULTILINE | re.DOTALL)
+assert m, "Can't find table"
+
+d = m[1].replace('\n', '')
+
+header = sc.find("<thead>\s*<tr>\s*(<th .*><strong>Datum</strong></th>\s*<th .*><strong>Fälle</strong><br />positiv</th>\s*<th .*>Im<br /><strong>Spital</strong><br />gesamt</th>\s*<th .*>Davon<br />normale<br /><strong>Betten-<br /> station</strong></th>\s*<th .*>Davon<br /><strong>Intensiv-<br /> station</strong><br />gesamt</th>\s*<th .*>Davon<br />Intensiv-<br />pflege<br /><strong>beatmet</strong></th>\s*<th .*><strong>Todes-<br /> fälle</strong></th>)\s*</tr>\s*</thead>", d)
```
Comment (metaodi) [inline scrapers/scrape_be.sh]: Use non-greedy matching for the `<th>` tag, like this: `<th.*?>`

Comment (calmyournerves) [reply scrapers/scrape_be.sh]: Done!

**Comment by baryluk** [comment]
Hi, @calmyournerves thanks a lot for the fix!

For the next time tho. The comment string in the source, should not be changed. Even if the content of the website changed. Each comment string is unmodified source content at a specific date. If you want to document a new format, you should make a new comment string, with a small note at which day it was retrived.

**Comment by baryluk** [comment]
Good fix! Nice.

**Comment by calmyournerves** [comment]
Got it @baryluk, thanks!

**Pull request merged**